core/src/main/java/io/kestra/core/models/executions/MetricEntry.java

```diff
@@ -36,6 +37,7 @@ public class MetricEntry implements DeletedInterface {
     String name;
 
     @NotNull
+    @JsonInclude
```

Comment:
By default Jackson include all fields right ? So it didn't seems necessary.

Reply:
Jackson remove the `0` value, that led to NaN on the front because it' was removed by the webserver

Reply:
Even for primitive wrapper ! This is a strange idea :(
OK thn.